package.json

```diff
@@ -27,7 +27,9 @@
     "build": "node bin/build",
     "coverage": "istanbul -- cover ./node_modules/.bin/nodeunit tests/unit",
     "data": "node scripts/generate-identifier-data",
-    "pretest": "jshint src && jscs src",
+    "jscs": "jscs src",
+    "jshint": "node bin/jshint src",
```

Comment:
I've glanced through the other PR's from you and I cannot see a connection.

> It's at least stupid to have jshint as a
> dependency for its own code.

You cannot just say something is stupid and a justification. I've provided an example where the current setup is *more* useful than your proposed setup. Why is your proposed setup more useful to people developing jshint?

> Again you clearly miss the point.

Sorry if I have, I'm just trying to sort out all the PR's, understand and review them. Please either explain within the PR, link to an explanation or you will have to rely on someone spending the time to try and understand what you meant, which is unlikely if you are not helpful and no benefit is obvious.